content/pages/about.md

```diff
@@ -41,4 +44,8 @@ do most of my work from the terminal, so why not add blogging too.
 ## DISCLAIMER ##
 Please keep this in mind while reading the blog:
 
-_Stuff just is. My mental model of how the world works is incomplete. It has boundaries. I'm constantly relearning stuff. That, there, is the thing. Knowing that I'm limited in how I perceive stuff. There is strength in these "limits" though, they enable you to filter noise and make decisions. ~ Bonface Munyoki_
+_Stuff just is. My mental model of how the world works is incomplete. It
+has boundaries. I'm constantly relearning stuff. That, there, is the
+thing. Knowing that I'm limited in how I perceive stuff. There is
+strength in these "limits" though, they enable you to filter noise and
+make decisions. ~ Bonface Munyoki_
```

Comment:
NIT: Lolz. So I'm reading a quote of myself, but it feels poorly structured :sweat: